sopel/irc/__init__.py

```diff
@@ -52,8 +50,8 @@
 from sopel import tools, trigger
```

Comment:
```suggestion
from sopel import tools, trigger
from sopel.lifecycle import deprecated
```

Goes with newly deprecated properties in the `AbstractBot`.